app/Filament/Pages/Settings/NotificationPage.php

```diff
@@ -175,6 +176,48 @@ public function form(Form $form): Form
                                         'default' => 1,
                                         'md' => 2,
                                     ]),
+                                
```

Comment:
Can you move the section up to below "Database", want to keep these in alphabetical order.